pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -435,6 +435,21 @@ func doUpdateVol(ctx *volumemgrContext, status *types.VolumeStatus) (bool, bool)
 			status.Key(), status.DisplayName)
 		return false, true
 	}
+
+	verifyOnly := false
+	vrc := lookupVolumeRefConfig(ctx, status.Key())
+	if vrc != nil {
+		verifyOnly = vrc.VerifyOnly
+	} else {
+		// check should we wait for ref config
+		vc := lookupVolumeConfig(ctx, status.Key())
+		if vc != nil && !vc.HasNoAppReferences {
+			log.Functionf("doUpdateVol(%s) name %s has app references but no VolumeRefConfig found",
+				status.Key(), status.DisplayName)
+			return false, true
```

Comment:
Why are you bailing in this case? 
This is the normal case when there is a race and you see the VolumeConfig before the VolumeRefConfig, thus it makes sense to set verifyOnly=true for now and later there will be a VolumeRefConfig.

Reply:
Good point, modified